application/classes/Ushahidi/Core.php

```diff
@@ -670,6 +689,19 @@ public static function init()
 		$di->setter['Ushahidi_Listener_PostSetListener']['setRepo'] =
 			$di->lazyGet('repository.notification.queue');
 
+		// Event listener for the Post repo
+		$di->setter['Ushahidi_Repository_Post']['setEvent'] = 'PostCreateEvent';
+		$di->setter['Ushahidi_Repository_Post']['setListener'] =
```

Comment:
The weirdest thing about this event/listener set up is that there can only be 1 listener.
Probably something we can address when integrating laravel, but there should really be 1 central event bus with unlimited listeners.